service/audit-service/src/main/java/uk/gov/gchq/palisade/audit/service/request/AuditRequestReceived.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.audit.service.request;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import uk.gov.gchq.palisade.Context;
+import uk.gov.gchq.palisade.ToStringBuilder;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * This is one of the objects that is passed to the {@link uk.gov.gchq.palisade.audit.service.AuditService}
+ * to be able to store an audit record. This class extends {@link AuditRequest} This class
+ * is used for the indication to the Audit logs that a processing request has been received.
+ */
+public class AuditRequestReceived extends AuditRequest {
+
+    public AuditRequestReceived() {
+    }
+
+    @Override
+    public String constructAuditLog() {
+        final String msg = "AuditRequestReceived: " + getContext().toString();
```

Comment:
Can you also get a request id (unique per request for data from the user and used to track the flow through the palisade services) the UserId from the request and the Resource alias/path so they can both be included in this audit log